apps/browser/src/key-management/session-timeout/services/extension-session-timeout-settings-component.service.ts

```diff
@@ -0,0 +1,58 @@
+import { defer, Observable, of } from "rxjs";
+
+import {
+  VaultTimeout,
+  VaultTimeoutOption,
+  VaultTimeoutStringType,
+} from "@bitwarden/common/key-management/vault-timeout";
+import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+import { MessagingService } from "@bitwarden/common/platform/abstractions/messaging.service";
+import { PlatformUtilsService } from "@bitwarden/common/platform/abstractions/platform-utils.service";
+import { SessionTimeoutSettingsComponentService } from "@bitwarden/key-management-ui";
+
+export class ExtensionSessionTimeoutSettingsComponentService extends SessionTimeoutSettingsComponentService {
```

Comment:
```suggestion
export class ExtensionSessionTimeoutSettingsComponentService implements SessionTimeoutSettingsComponentService {
```

These should just implement the abstraction. I don't think there is any reason for inheritance.

Reply:
Fixed in a10bf02c